task-launcher/src/tasks/trog/helpers/scores.js

```diff
@@ -51,12 +51,14 @@ export const computedScoreCallback = (rawScores) => {
   // computedScores should now have keys for each subtask.
   // But we also want to update the total score so we add up all of the others.
   //const totalScore = _reduce(_omit(computedScores, ['composite']), (sum, score) => sum + score.subScore, 0);
-  const { totalCorrect, trialNumTotal } = taskStore();
+  const { totalCorrect } = taskStore();
 
   computedScores.composite = {
     totalCorrect: totalCorrect,
-    totalNumAttempted: trialNumTotal,
-    totalPercentCorrect: Math.round((totalCorrect / trialNumTotal) * 100),
+    totalNumAttempted: rawScores?.composite?.test?.numAttempted,
```

Comment:
Is this functionality only being used by ROAR?

Reply:
Yes, just for Roar apps. It was writing infinity to our database, which gave us an error on Redivis. The actual data is the one on raw